internal/dwarf/frame/table.go

```diff
@@ -401,9 +413,8 @@ func lookupFunc(instruction byte, ctx *Context) instruction {
 	case DW_CFA_GNU_window_save:
 		fn = gnuwindowsave
 	default:
-		panic(fmt.Sprintf("Encountered an unexpected DWARF CFA opcode: %#v", instruction))
+		level.Error(ctx.logger).Log("msg", "encountered an unexpected DWARF CFA opcode", "opcode instruction", instruction, "PID", getPid(ctx.fullExecutablePath), "comm", ctx.fullExecutablePath)
```

Comment:
I would prefer if we didn't pass the logger around like this and instead have this function return an error that we then pass up and layer on context as we pass the error further up.